src/scss/components/paragraphs/_chart.scss

```diff
@@ -0,0 +1,25 @@
+$_chart-aspect-ratio: 1.78; // 16/9
+
+.chart {
+  figure {
+    // Reset user-agent stylesheet rules for figure.
+    margin: 0;
+  }
+}
+
+.chart__container {
+  height: 0;
+  margin: $spacing 0;
```

Comment:
If you're not setting margin-left and -right to 0 for overriding some other margin, please split this to multiple properties.